src/status_im/ui2/screens/chat/composer/input.cljs

```diff
@@ -137,13 +137,28 @@
     (when platform/android?
       (>evt [::mentions/calculate-suggestions mentionable-users]))))
 
+(defn force-text-input-update!
+  "force-text-input-update! forces the
+  input to re-render, necessary when we are setting value"
+  []
+  (swap! chat-input-key inc))
+
+(re-frame/reg-fx
+ :set-input-text
+ (fn [[chat-id text]]
+    ;; We enable mentions
```

Comment:
I think the code is very self-explanatory that we don't need these comments, but it might be just me.